src/main/java/org/ow2/proactive/catalog/repository/BucketRepository.java

```diff
@@ -62,16 +61,17 @@
     @Query(value = "SELECT bk FROM BucketEntity bk WHERE bk.bucketName = ?1")
     BucketEntity findBucketForUpdate(String bucketName);
 
-    @Query(value = "SELECT bk.bucketName, bk.owner, COUNT(cos.id.name) as objectCount FROM BucketEntity bk LEFT JOIN bk.catalogObjects cos" +
+    @Query(value = "SELECT bk.bucketName, bk.owner, COUNT(cos.id.name) as objectCount, bk.id FROM BucketEntity bk LEFT JOIN bk.catalogObjects cos" +
```

Comment:
This will cause `ORA-00979: not a group by expression` exception https://www.tekstream.com/resource-center/ora-00979-not-a-group-by-expression/. You should have `bk.id` in the SELECT clause.

Reply:
Thanks. I know about this issue. That's why I've added bk.id in the SELECT clause.
`SELECT bk.bucketName, bk.owner, COUNT(cos.id.name) as objectCount, bk.id`
Do you see it?

Reply:
Sorry I mean in the "group by"

Reply:
You have only `GROUP BY bk.bucketName, bk.owner`

Reply:
Actually I was thinking about other issue related to Select and Order By.
Did you mean that bk.id should be added in Group by?
GROUP BY bk.bucketName, bk.owner, bk.id

Reply:
Yes